axis_backend/apps/kpis/services/kpi_service.py

```
@@ -0,0 +1,288 @@
+    @transaction.atomic
+    def create_kpi(
+        self,
+        name: str,
+        type_id: str,
+        unit: str,
+        unit_type: str,
+        description: Optional[str] = None,
+        target_value: Optional[str] = None,
+        calculation_method: Optional[str] = None,
+        frequency: Optional[str] = None,
+        is_public: bool = True,
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)